cmd/immuclient/immuc/login.go

```diff
@@ -173,7 +173,7 @@ func (i *immuc) ChangeUserPassword(args []string) (string, error) {
 	if err := i.ImmuClient.ChangePassword(context.Background(), []byte(username), oldpass, []byte(newpass)); err != nil {
 		return "", err
 	}
-	return fmt.Sprintf("Password of %s was changed successfuly", username), nil
+	return fmt.Sprintf("Password of %s was changed successfully", username), nil
```

Comment:
hi @muesli, probably "Password of %s was successfully changed" it's better. If so, could you please change it and we can merge your PR. Thanks a lot

Reply:
Sure thing. Done!